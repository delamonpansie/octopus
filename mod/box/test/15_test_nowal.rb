#!/usr/bin/ruby

$: << File.dirname($0) + '/lib'
require 'run_env'

class Env < RunEnv
  def config
    super + <<EOD
wal_writer_inbox_size=0
EOD
  end
end

Env.connect_eval do
  12.times {|i| insert [i,i]}
  12.times {|i| select i}
  puts Dir.glob("*.xlog").sort.join("\n")
end
