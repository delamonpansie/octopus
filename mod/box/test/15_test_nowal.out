# box.insert([0, 0])
1

# box.insert([1, 1])
1

# box.insert([2, 2])
1

# box.insert([3, 3])
1

# box.insert([4, 4])
1

# box.insert([5, 5])
1

# box.insert([6, 6])
1

# box.insert([7, 7])
1

# box.insert([8, 8])
1

# box.insert([9, 9])
1

# box.insert([10, 10])
1

# box.insert([11, 11])
1

# box.select(0)
[["\x00\x00\x00\x00", "\x00\x00\x00\x00"]]

# box.select(1)
[["\x01\x00\x00\x00", "\x01\x00\x00\x00"]]

# box.select(2)
[["\x02\x00\x00\x00", "\x02\x00\x00\x00"]]

# box.select(3)
[["\x03\x00\x00\x00", "\x03\x00\x00\x00"]]

# box.select(4)
[["\x04\x00\x00\x00", "\x04\x00\x00\x00"]]

# box.select(5)
[["\x05\x00\x00\x00", "\x05\x00\x00\x00"]]

# box.select(6)
[["\x06\x00\x00\x00", "\x06\x00\x00\x00"]]

# box.select(7)
[["\a\x00\x00\x00", "\a\x00\x00\x00"]]

# box.select(8)
[["\b\x00\x00\x00", "\b\x00\x00\x00"]]

# box.select(9)
[["\t\x00\x00\x00", "\t\x00\x00\x00"]]

# box.select(10)
[["\n\x00\x00\x00", "\n\x00\x00\x00"]]

# box.select(11)
[["\v\x00\x00\x00", "\v\x00\x00\x00"]]


